src/test/java/pages/LoginPage.java

```diff
@@ -4,6 +4,38 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
+
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class LoginPage extends BasePage{
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/225231155-0ac1a737-0b4a-4ec1-b1a2-3be87a8dbd0a.png)

Reply:
I run again all 5 tests in gradle and all passed. I do not understand were the issue is?
![image](https://user-images.githubusercontent.com/125710494/225422559-5cab6de9-b91b-43f7-a998-f4823d81fac3.png)

Reply:
Please re-submit homework 23 for review

Reply:
Hi Alex, I did send a message in LMS were I submit the link. My Hm is destroyed, because all the lines of code added by Khaled on Monday's class has just populated on my homework, How to fixed it

Reply:
Something was done on the repository with Khaled's work and  it did affect my homework. Please someone help me.